socket-server/internal/ws/server.go

```
@@ -33,38 +38,139 @@ func RunServer(w http.ResponseWriter, r *http.Request) {
 
-	userId, err := validation.ValidateSupabaseJWT(token, os.Getenv("SUPABASE_JWT_KEY"))
+	userId, err := validation.ValidateSupabaseJWT(token, config.EnvVars.SupabaseJwtKey)
 	if err != nil {
 		http.Error(w, "invalid token", 401)
 		log.Println("Invalid token:", err)
 		return
 	}
 
+	targetLanguage := r.URL.Query().Get("target_language")
+	if !translation.IsLanguageSupported(targetLanguage)  {
+		http.Error(w, "Language is not supported yet.", 401)
```

golangci-lint (2.12.2)

[error] ST1013: should use constant http.StatusUnauthorized instead of numeric literal 401

(staticcheck)